### ISSUE TITLE
Installing into project using react_webpack_rails fails

### ISSUE THREAD

**Issue by rhulse** [sstur/react-rte#34]
The were a bunch of dependencies that could not be met. I did an npm update and got the same result.

```
$ npm install --save react-rte
npm ERR! Darwin 15.4.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "--save" "react-rte"
npm ERR! node v0.10.36
npm ERR! npm  v2.3.0
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package react does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer draft-js@0.4.0 wants react@^0.14.0 || ^15.0.0-rc
npm ERR! peerinvalid Peer react-addons-css-transition-group@0.14.8 wants react@^0.14.8
npm ERR! peerinvalid Peer react-dom@0.14.8 wants react@^0.14.8
npm ERR! peerinvalid Peer react-webpack-rails@0.3.1 wants react@^0.14.0
npm ERR! peerinvalid Peer react-rte@0.1.13 wants react@^15.0.1
```

**Comment by ZeroCho** [comment]
same issue happens to me..
I think @sstur  should consider upgrading draft-js dependency version of react-rte in package.json to ^0.5.0

**Comment by sstur** [comment]
Oh, I'll try that now. Thanks.

**Comment by sstur** [comment]
I think this was fixed with v0.1.15 so I will close this, but let me know if the problem persists. Thx
